mirdata/jams_utils.py

```diff
@@ -0,0 +1,257 @@
+# -*- coding: utf-8 -*-
+"""functions for converting mirdata annotations to jams format
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import jams
+import mirdata.utils as utils
+
+def jams_converter(
+        beat_data=None,
+        chord_data=None,
+        note_data=None,
+        f0_data=None,
+        lyrics_data=None,
+        section_data=None,
+        multi_section_data=None,
+        key_data=None,
+        metadata=None):
+
+    """Convert annotations to JAMS format.
+
+    Args:
+        beat_data (list or None):
+            A list of tuples of (BeatData, str), where str describes the annotation (e.g. beats_1).
+        chord_data (list or None):
+            A list of tuples of (ChordData, str), where str describes the annotation.
+        note_data (list or None):
+            A list of tuples of (NoteData, str), where str describes the annotation.
+        f0_data (list or None):
+            A list of tuples of (F0Data, str), where str describes the annotation.
+        section_data (list or None):
+            A list of tuples of (SectionData, str), where str describes the annotation.
+        multi_section_data (list or None):
+            A list of tuples. Tuples in multi_section_data should contain another
+            list of tuples, indicating annotations in the different levels
+            e.g. ([(segments0, level0), '(segments1, level1)], annotator) and a str
+            indicating the annotator
+        key_data (list or None):
+            A list of tuples of (KeyData, str), where str describes the annotation.
+        lyrics_data (list or None):
+            A list of tuples of (LyricData, str), where str describes the annotation.
+        metadata (dict or None):
+            A dictionary containing the track metadata.
+    """
+
+    jam = jams.JAMS()
+
+    # metadata
+    if metadata is not None:
+        if 'duration_sec' in metadata.keys():  # match name to jams attribute
+            metadata['duration'] = metadata.pop('duration_sec')
+        if 'Title' in metadata.keys():  # match name to jams attribute
+            metadata['title'] = metadata.pop('Title')
+        if 'Artist' in metadata.keys():  # match name to jams attribute
+            metadata['artist'] = metadata.pop('Artist')
```

Comment:
I think this bit doesn't belong here - any logic converting a dataset's specific metadata should probably happen in the dataset's `to_jams` function, and this function can blindly add keys when they match. But, we should probably document which keys are supported.

Reply:
Yeah, makes sense. It was a sanity check I added. I'll remove it and stand in the docs that `to_jams()` supports `jams` metadata keys